libraries/AP_BoardConfig/AP_BoardConfig.h

```diff
@@ -132,6 +132,10 @@ class AP_BoardConfig {
     uint16_t get_safety_button_options(void) {
         return uint16_t(state.safety_option.get());
     }
+
+    bool get_safety_enable(void) {
```

Comment:
`is_safety_enabled(void) const` might be a better name? I'd normally thing of get as an accessor, for a case like this, where you really want it as just a boolean.

Reply:
Okay more importantly this function isn't actually used anywhere?

Reply:
it was used in my various uncommitted versions of this PR, but isn't now. I've removed it